scripts/upgrade

```diff
@@ -71,19 +71,19 @@ fi
 #=================================================
 
 [[ $YNH_APP_ID == "garradin" ]] \
-    && [[ "$(cat "/opt/$app/templates/.VERSION")" != 1.2.2 ]] \
+    && [[ "$(cat "/$final_path/VERSION")" != 1.2.2 ]] \
```

Comment:
```suggestion
    && [[ "$(cat "$final_path/VERSION")" != 1.2.2 ]] \
```

Reply:
thanks for the review...

Reply:
I miss the command `git log -p`  because I have commit your change first... but I push the changes... Also a check_process to test garradin migration...